tapdance/assets.go

```diff
@@ -137,6 +141,29 @@ func (a *assets) readConfigs() {
 		return nil
 	}
 
+	readTmpBackoff := func(filename string) error {
+		tmpBackoffStr, err := ioutil.ReadFile(filename)
+		if err != nil {
+			return err
+		}
+		backoffInt64, err := strconv.ParseInt(string(tmpBackoffStr), 10, 64)
+		if err != nil {
+			Logger.Errorln("Failed to Parse tmp_backoff:", err)
+			return err
+		}
+		if time.Now().Unix() > backoffInt64 {
+			err = os.Remove(filename)
```

Comment:
This function appears to implement "there is no longer a backoff" as "remove the backoff file", but also treats the lack of a backoff file as an error.